sympy/series/limits.py

```diff
@@ -109,6 +109,22 @@ def __new__(cls, e, z, z0, dir="+"):
         obj._args = (e, z, z0, dir)
         return obj
 
+
+    @property
+    def free_symbols(self):
+        e = self.args[0]
+        # e and z are same then none is free_symbols
+        if e is self.args[1]:
+            isyms = set()
+        else:
+            isyms = e.free_symbols
+            # take out z from isyms
+            if self.args[1] in isyms:
+                isyms.remove(self.args[1])
```

Comment:
Yes i think. Because `Limit(x**2 + y**4, x**2 + y**4, a).free_symbols` should return `set([a])` ( i have included a similar simple expression in `test cases` ). Since we have `Limit(expr, expr, a).doit()` returns the `a` , which is independent of `x` and `y` basically.